templates/default/thin/init.d.erb

```diff
@@ -25,7 +25,7 @@ check_run_dir() {
 		mkdir /var/run/thin
 	fi
 	chown redmine:redmine /var/run/thin
-	chmod 0755 /var/run/thin
+	chmod 0750 /var/run/thin
```

Comment:
Actually, that's not sufficient, as nginx can't access the socket then. 0755 or set the group to nginx?